book/06-github/sections/3-maintaining.asc

```diff
@@ -330,11 +330,11 @@ If GitHub sees a README file in your source, it will render it on the landing pa
 Many teams use this file to hold all the relevant project information for someone who might be new to the repository or project.
 This generally includes things like:
 
-* What the project is for
-* How to configure and install it
-* An example of how to use it or get it running
-* The license that the project is offered under
-* How to contribute to it
+* What the project is for.
+* How to configure and install it.
+* An example of how to use it or get it running.
+* The license that the project is offered under.
+* How to contribute to it.
```

Comment:
I'd say leave the periods off of these. It's a bullet list of sentence fragments.

Reply:
Actually, I follow [Garner's](https://en.wikipedia.org/wiki/Garner%27s_Modern_English_Usage) book for punctuation, in which it's stated that if bullet point starts with a lowercase letter, end it with semicolon except last one which ends with period. And if bullet starts with uppercase letter end it with period; and by convention a fragment would be acceptable units. Let me check it for APA style.

Reply:
Okay, in APA style we may not use punctuation with fragments of sentence; but my question is, are these statements really fragments of a sentence or a complete sentence? These statements can exist alone; and also if we're assuming these statements as fragments, it should start with lowercase.